### PR TITLE
Adds btn.basic style

### PR DESCRIPTION
**Pull request by scott-joe**
Fixes https://github.com/ExactTarget/fuelux-mctheme/issues/336

### CODE REVIEW

File: less/bootstrap-override/buttons.less [hunk 1]
```diff
@@ -104,6 +102,56 @@
...
+	&:active,&.active,&:focus,&:active:focus,&:active:hover {
+		#gradient > .gradient-bg(@btn-basic-bg-press-start, @btn-basic-bg-press-end);
+		color: @btn-basic-color;
+	}
+
+	&.dropdown-toggle .caret {
+		color: @btn-basic-color;
+	}
+
+	&.btn-lg {
```
Comment (swilliamset) [inline less/bootstrap-override/buttons.less]: can the `.btn-lg` `.btn-sm` `.btn-xs` portions be dried up? is the padding different here than on other button types?

**Comment by scott-joe** [comment]
The syntax and formatting is identical to the `.btn-default` class. https://github.com/scott-joe/fuelux-mctheme/blob/master/less/bootstrap-override/buttons.less#L64 I was just following the pattern. Unless I'm not understanding your request.

**Comment by swilliamset** [comment]
just wondered if there was some repeated code that could be moved to a mixin

otherwise @scott-joe please update your branch

**Comment by scott-joe** [comment]
That structure is in alignment with the rest of the button code. A refactor of the button LESS to become more DRY through the introduction of other mixins would be something I'd defer to true maintainers of the theme.

I've merged in new changes and removed the `/dist` dir from my commit

**Comment by interactivellama** [comment]
I'm good with this if the designers are.

For reference:
screenshot 2015-08-18 15 27 07

screenshot 2015-08-18 15 27 16

**Comment by mbeard** [comment]
Can you confirm that the font color used for the gray on gray button is consistent with our font color on selectlist too?

**Comment by scott-joe** [comment]
It looks like there are tiny variations in the colors used. It took some pulling from about 4 files but this is what I distilled it down to:

```@gray93: lighten(#000, 93.5%);   // @EEE

*Select List:*
----------------------------
@selectableHover: #f1f1f1;
@selected: #efefef;
@selectedHover: #d7d7d7;
@disabled-background: #fcfcfc;
@border-color: #ddd;

background: linear-gradient(#FFFFFF, @gray93);
border: 1px solid #bbb;

.caret {
	color: #656565;
}

.selected-label {
	color: #656565;
}

*Basic Button:*
----------------------------
.btn-basic {
	border: 1px solid #bbb;
	#gradient > .gradient-bg(#FEFEFE,#E0E0E0);

	&:active,&.active,&:focus{
		background: @gray93;
		.box-shadow(inset 0 1px 1px rgba(0,0,0,.2));
	}

	.button-variant(#000;
					#e0e0e0; #e0e0e0;
					#F4F1F1; #CBCACA;
					#EAEAEA; #F6F6F6;
					#FFF; #D7D7D7);
	&:active,&.active,&:focus,&:active:focus,&:active:hover {
		#gradient > .gradient-bg(#EAEAEA, #F6F6F6);
		color: #000;
	}

	&.dropdown-toggle .caret {
		color: #000;
	}
}```

**Comment by interactivellama** [comment]
@mbeard Thanks. Dropdowns appear to be styled by the `body` color attribute. 

@scott-joe The button is #000 it should not be specified and be the `body` color I think. The MCtheme body color is `#333`.

It would be nice to make sure `.btn-dropdown` and this use the same colors. or btn-dropdown could use `.btn-basic()`.

**Comment by scott-joe** [comment]
So, there are some subtle but distinct differences between the Selectlist button and the default button. I've done my best to blend the two for visual similarity while remaining contextually honest to their grouping. I made sure all the colors were correct, but the biggest difference is that the selectlist button does not "go dark" when :active. It simply changes it's box-border to an inset and that's it. Our buttons all handle :active differently, but since I'm basing this on btn.default, I basically took it's :active bkg-gradient from HEX to HSL, dropped the S to 0% and converted it back to HEX, creating a mono-chrome version of the color range seen in btn.default. So, it looks like a Selectlist when not selected, but behaves like btn.default when clicked...

Normal:
screen shot 2015-08-18 at 4 40 24 pm
screen shot 2015-08-18 at 4 51 18 pm

Active:
screen shot 2015-08-18 at 4 39 57 pm
screen shot 2015-08-18 at 4 51 29 pm

Disabled:
screen shot 2015-08-18 at 4 39 23 pm
screen shot 2015-08-18 at 4 51 39 pm

**Comment by scott-joe** [comment]
At this point, if the differences are still unacceptable, I would recommend making the selectlist button more like, and even abandon it's unique stylings in favor of simply applying a `.basic` tag to the official selectlist markup.

**Comment by swilliamset** [comment]
please update branch to resolve merge conflict

File: less/bootstrap-override/type.less [hunk 2]
```diff
@@ -1,4 +1,4 @@
-/*
+ll/*
```
Comment (swilliamset) [inline less/bootstrap-override/type.less]: fix it! :+1:

File: less/bootstrap-override/variables.less [hunk 3]
```diff
@@ -93,6 +93,17 @@
...
+@btn-basic-color:              #656565;
+@btn-basic-bg-start:           #fefefe;
+@btn-basic-bg-end:             #e0e0e0;
+@btn-basic-bg-hover-start:     #F4F1F1;
+@btn-basic-bg-hover-end:       #CBCACA;
+@btn-basic-bg-press-start:     #E0E0E0;
+@btn-basic-bg-press-end:       #EBEBEB;
+@btn-basic-bg-disabled-start:  #FFF;
+@btn-basic-bg-disabled-end:    #D7D7D7;
+@btn-basic-border:             #bbb;
```
Comment (swilliamset) [inline less/bootstrap-override/variables.less]: revisit variables to see if all these are needed and if there are any color pallet variables that could be mapped to. #299

**Comment by swilliamset** [comment]
wait for #299

**Comment by scott-joe** [comment]
Switched to use new `@grayxx` values. Thanks for pointing this out. Also fixed a thing or two in the class itself. also added a little script to make the color blocks more readable.

**Pull request merged**